NOTICE.txt

```diff
@@ -0,0 +1,3980 @@
+Included Software, Images and Icons
```

Comment:
Please make this a markdown file and adhere to markdown conventions. You're using something like markdown (or rst?) but it's clear it's none of them.

Reply:
Sorry, which markdown or rst-like conventions have I used? I'm all about following standard formats, but this file already does conform to a very basic one, the standard for legal notices in software such as this: plain text. The only formatting/markup I employed that I'm aware of that's similar to either Markdown or rst is the underlines for the headings like in rst (as you've highlighted here), which is often seen in plain text files as well for long before either format existed, and simply serves to unambiguously distinguish the heading levels. Everything else is just plain text.

Furthermore, there are a number of reasons why either Markdown or reST wouldn't make much sense for this file, or even actively detract from it:

* Markdown and reST both disregard most line breaks and display engines soft-wrap text; whereas here line breaks and their number are often quite significant at clearly, consistently and unambiguously communicating the required information, which is paramount above all else for this document in particular.
* They would not necessarily preserve verbatim all of the characters and plain text formatting present in the original legal notices, disclaimers, copyright statements, and attributions that I had transcended from the original project sources, which most licenses require to varying degrees of explicitness.
* For maximum interoperability, minimum ambiguity, and to avoid any chance of a parser affecting the letter nor the spirit, legal files such as this are almost always plain text
* Fancy rendering and pretty formatting accomplish little or nothing or are even counterproductive for a legal document intended to plainly, clearly and ambiguously communicate information and fulfill specific verbatim requirements
* NOTICE has to be either without an extension or ``.txt`` per Apache 2.0 license requirements, of which two of the projects we include (MathJax and Githubpy) have as their license.
* Given that other than the reST-esque headings, the 4000-line document employs no Markdown or reST formatting of any kind nor anything similar, I don't see any meaningful benefits other than an unnecessary pretty veneer that would justify the effort needed to manually apply such that could be better spent elsewhere; even notwithstanding all the issues mentioned above

Reply:
Ok, let's leave it as plain text. Perhaps what I don't like is that there are a lot of blanks at the beginning. Please try to remove them.